src/Samhammer.Authentication.Api/Keycloak/KeycloakClaimsTransformation.cs

```diff
@@ -66,20 +67,17 @@ private void MapRoleClaim(ClaimsIdentity claimsIdentity)
             }
         }
 
-        private void MapNameClaim(ClaimsIdentity claimsIdentity)
+        public void MapNameClaim(ClaimsIdentity claimsIdentity)
         {
-            if (claimsIdentity.HasClaim(c => c.Type == ClaimTypes.Name))
+            if (claimsIdentity.TryGetClaim(c => c.Type == AuthOptions.Value.NameClaim, out var claimToSet))
```

Comment:
@ckittelmann Up to now, we kept "ClaimTypes.Name"  if it was already set, so that this was the default with a fallback to "preferred_username". Now the default is "preferred_username".

This change is ok, but we may mention it in the changelog because some apps might need an additional change on upgrading.
The property "NameClaim" needs to be set to "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name" to keep those applications functional.